packages/stytch-auth/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@hono/stytch-auth",
+  "version": "0.0.0",
+  "description": "A third-party Stytch auth middleware for Hono",
+  "type": "module",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsup ./src/index.ts",
+    "prepack": "yarn build",
+    "publint": "attw --pack && publint",
+    "typecheck": "tsc -b tsconfig.json",
+    "test": "vitest"
+  },
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
+    }
+  },
+  "license": "MIT",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org",
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/honojs/middleware.git",
+    "directory": "packages/stytch-auth"
+  },
+  "homepage": "https://github.com/honojs/middleware",
+  "peerDependencies": {
+    "hono": ">=3.*",
+    "stytch": "^12.19.0"
+  },
+  "devDependencies": {
+    "@arethetypeswrong/cli": "^0.17.4",
+    "@types/react": "^18",
+    "publint": "^0.3.9",
+    "react": "^18.2.0",
```

Comment:
Are these `react` related libs necessary?

Reply:
Oop, nope. Copy/paste from when I created the package. Fixed in https://github.com/honojs/middleware/pull/1186/commits/3d5636f1e5dfc6ef184cc53bbfea934215fdf226

Reply:
Thanks!